packages/react-i18n/src/hooks.tsx

```diff
@@ -24,7 +24,7 @@ export function useI18n({
   const parentIds = parentI18n ? parentI18n.ids || [] : [];
   const ids = React.useMemo(() => (id ? [id, ...parentIds] : parentIds), [
     id,
-    ...parentIds,
+    parentIds,
```

Comment:
No, please keep the spread. This is not equivalent and means it will recompute this when there is no parentI18n.